tests/Unit/PathTest.php

```diff
@@ -15,12 +15,27 @@
 
 use OpenCFP\Environment;
 use OpenCFP\Path;
+use OpenCFP\PathInterface;
 
 /**
  * @covers \OpenCFP\Path
  */
 final class PathTest extends \PHPUnit\Framework\TestCase
 {
+    public function testIsFinal()
+    {
+        $reflection = new \ReflectionClass(Path::class);
+
+        $this->assertTrue($reflection->isFinal());
+    }
+
+    public function testImplementsPathInterface()
+    {
+        $reflection = new \ReflectionClass(Path::class);
+
+        $this->assertTrue($reflection->implementsInterface(PathInterface::class));
```

Comment:
It looks like we either test this with ``` $reflection->implementsInterface()``` or with ```assertInstanceOf()```, may be best to standardise it to either one.